site/en/blog/media-updates-in-chrome-69/index.md

```diff
@@ -0,0 +1,289 @@
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/MeaweyrpR2iIVbXZqemN.png", alt="PTS/DTS", width="800", height="434" %}
+  <figcaption>
+    PTS/DTS
+  </figcaption>
+</figure>
+
+This change affects `MediaSource.duration` (and consequently
+`HTMLMediaElement.duration`), `SourceBuffer.buffered` (and consequently
+`HTMLMediaElement.buffered)`, and `SourceBuffer.remove(start, end)`.
+
+If you're not sure which method is used to report buffered ranges and duration
+values, you can go to the internal `chrome://media-internals` page and search for
+"ChunkDemuxer: buffering by PTS" or  "ChunkDemuxer: buffering by DTS" in the
+logs.
+
+[Intent to Implement](https://groups.google.com/a/chromium.org/d/msg/blink-dev/I6oGJLym-Tk/Z46le9l4CQAJ) &#124;
+[Chromium Bug](https://crbug.com/718641)
+
+## Handling of media view intents on Android Go {: #media_intent_handler }
+
+[Android Go] is a lightweight version of Android designed for entry-level
+smartphones. To that end, it does not necessarily ship with some media-viewing
+applications, so if a user tries to open a downloaded video for instance, they
+won't have any applications to handle that intent.
+
+To fix this, Chrome 69 on Android Go now listens for media-viewing intents so
+users can view downloaded audio, videos, and images. In other words, it takes
+the place of the missing viewing applications.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/qGcEJcsQz6oRpoqE8mCE.png", alt="ALT_TEXT_HERE", width="800", height="369" %}
+  <figcaption>
+    Media intent handler
+  </figcaption>
+</figure>
+
+Note that this Chrome feature is enabled on all Android devices running Android
+O and onwards with 1 GB of RAM or less.
+
+[Chromium Bug](https://crbug.com/718641)
+
+## Removal of "stalled" events for media elements using MSE {: #stalled }
+
+A "stalled" event is raised on a media element if downloading media data has
+failed to progress for about 3 seconds. When using [Media Source Extensions
+(MSE)], the web app manages the download and the media element is not aware of
+its progress. This caused Chrome to raise "stalled" events at inappropriate
+times whenever the website has not appended new media data chunks with
+`SourceBuffer.appendBuffer()` in the last 3 seconds.
+
+As websites may decide to append large chunks of data at a low frequency, this
+is not a useful signal about buffering health. Removing "stalled" events for
+media elements using MSE clears up confusion and brings Chrome more in line
+with the MSE specification. Note that media elements that don't use MSE will
+continue to raise "stalled" events as they do today.
+
+[Intent to Deprecate and Remove](https://groups.google.com/a/chromium.org/d/msg/blink-dev/x54XtrTyOP8/4-5QZlZzDAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/6338037575319552) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=836951)
```

Comment:
This "[Chromium Bug]" link is included here and also on line 241.

1. Is our linter complaining at all?
2. What's the intended behavior?

Reply:
`[foo](https://a)` and `[foo](https://b)` are valid in the same document.
They're not definitions: `[foo]: https://a` and `[foo]: https://b`.